docs/src/blockchain-development/access_control.md

```diff
@@ -15,3 +15,17 @@ The `msg_sender` function works as follows:
 - If the caller is a contract, then `Result::Ok(Sender)` is returned with the `ContractId` sender variant.
 - If the caller is external (i.e. from a script), then all coin input owners in the transaction are checked. If all owners are the same, then `Result::Ok(Sender)` is returned with the `Address` sender variant.
 - If the caller is external and coin input owners are different, then the caller cannot be determined and a `Result::Err(AuthError)` is returned.
+
+## Using `BASE_ASSET_ID` as deafult/null address
+
+The `BASE_ASSET_ID` constant is **not** a reserved address and should not be treated as such.
+
+`BASE_ASSET_ID` is defined in the Standard Library. It is a `b256` type and is eqivalent to the 0 value.
+
+Many contracts require some form of ownership. When revoking ownership, it is disadvised to use the `BASE_ASSET_ID` of a `Address` or `ContractId` to reset the owner. Instead, it is recommended to use an `Option` of type `Identity` and setting the `Option` to `None`. 
```

Comment:
This makes less sense in the light of #2020

Reply:
For anyone coming from Solidity, it will be instinctive to set an address to 0 as a default. As `BASE_ASSET_ID` will become a `ContractId` it may make more sense to have this section refer to a 0 address in general rather than the constant itself.

Reply:
Ref: #2131

Reply:
I've updated this section to refer to contract ownership in general and the use of the zero address.